Buildings/Experimental/DHC/Loads/Cooling/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -0,0 +1,295 @@
+within Buildings.Experimental.DHC.Loads.Cooling.BaseClasses;
+model BuildingTimeSeries
+  "Building model with cooling load provided as time series"
+  extends Buildings.Experimental.DHC.Loads.BaseClasses.PartialBuilding(
+    redeclare package Medium=Buildings.Media.Water,
+    have_heaWat=false,
```

Comment:
should the two parameters be finalized?

Reply:
You are right, I have add the "final" prefix before these two parameters.